src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingNamedTypeSymbol.cs

```diff
@@ -8,9 +8,14 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Diagnostics;
+using System.Reflection.Metadata;
 using System.Runtime.CompilerServices;
 using System.Threading;
+using Microsoft.Cci;
```

Comment:
This using looks unnecessary if we use `Cci.` prefix in the code below.

Reply:
> This using looks unnecessary

IDE generated stubs for interface implementation and added this using in the process. That code doesn't use prefix.